client/src/FindSchool.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { Button, ButtonGroup } from "@chakra-ui/react";
+
+function FindSchool () {
+    return (
+        <div>
+        <div className="schoolForm">
```

Comment:
```suggestion
        <form className="schoolForm">
```
Can we use form instead of div?

Reply:
Done